main.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+)
+
+// This will be loaded from env later I'm assuming. For now just returns what port is used on localhost.
+const PORT string = ":8080"
+
+func main() {
+	fmt.Println("serving on port: ", PORT)
+	router := gin.Default()
+	router.GET("/health", getHealth)
+	http.ListenAndServe(PORT, router)
```

Comment:
the first argument to `ListenAndServe` is actually a full address, and it's best to always be able to specify the host too.  

I'd say either have another const for `HOST` and fmt them together to be `HOST:PORT` or have a const for `ADDRESS` that needs to be the fully host string. 

Kinda see it both ways in the wild, but probably a slight preference for the former.

Reply:
Should default to `127.0.0.1` so it's more secure.  The other common default is `0.0.0.0` but better to default to safe even if it's more annoying.

Reply:
PORT should still be used for "serving on:" correct? Or should it list both?

Reply:
I'm thinking:
```
const HOST string = "127.0.0.1"
const PORT string = "8080"

// down later
address = fmt.sprintf("%s:%s", HOST, PORT)
fmt.Println("serving on address: ", address)

http.ListenAndServe(address, router)

```